app/classifier/index.cjsx

```diff
@@ -40,6 +41,7 @@ Classifier = React.createClass
     subject: null
     classification: null
     onLoad: Function.prototype
+    cacheClassification: new CacheClassification
```

Comment:
`export const cachedClassification = new CacheClassification` should do the trick, I think. No big deal if it doesn't work, though.  https://developer.mozilla.org/en/docs/web/javascript/reference/statements/export